proteus/postprocessing.c

```diff
@@ -2637,6 +2638,9 @@ Input Variables
     nVDOFs_element,nSpace);
   */
 
+  /* printf("BDM1 test information: \n"); */
+  /* printf("nVDOFs_element: %d\n",nVDOFs_element); */
```

Comment:
I would only leave code in comments like this if it's complicated debugging code that is likely to get used again.